tests/integration/kubernetes/gha-run.sh

```diff
@@ -210,6 +210,13 @@ function deploy_kata() {
 		  "${NO_PROXY}"
 	fi
 
+	if [[ "${PULL_TYPE}" != "default" && -n "${PULL_TYPE}" ]]; then
```

Comment:
Do we need the first check? I'm thinking about the case we have a modified `kata-deploy.yaml` with something else, and I would like to bring back to default? Maybe just checking if `${PULL_TYPE}` is set to something?